.travis.yml

```diff
@@ -8,7 +8,7 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+    - CONDA_R=3.3.1
```

Comment:
@ocefpaf O.k., I'm going to use conda smithy the next time.

For the publication of our workflow we'd only need 3.3.1. Going to 3.3.2 is not an easy decision if you need to reprocess terabytes of data and -- much worse -- check the results to ensure things are staying the same. Given that there are a number of 3.3.1 packages in bioconda I would then resort to adding the package version there.

Thanks.